util/ranger/ranger_test.go

```diff
@@ -1443,6 +1443,8 @@ func (s *testRangerSuite) TestIndexRangeForBit(c *C) {
 	c.Assert(err, IsNil)
 	testKit := testkit.NewTestKit(c, store)
 	testKit.MustExec("use test;")
+	testKit.MustExec("set @@tidb_partition_prune_mode = 'static-only';")
+	testKit.MustExec("set @@tidb_executor_concurrency = 1;")
```

Comment:
The reason why we `set @@tidb_executor_concurrency = 1` is due to the concurrent execution of PartitionUnion, we need the results to stabilize.